roles/poa-logrotate/tasks/main.yml

```diff
@@ -3,4 +3,5 @@
   template: src=poa-logrotate.conf.j2 dest={{ home }}/poa-logrotate.conf owner={{ username }} group={{ username }} mode=0644
 
 - name: Install poa-logrotate cron
-  template: src=poa-logrotate.j2 dest=/etc/cron.hourly/poa-logrotate owner=root group=root mode=0755
+  cron: special_time=hourly job="/usr/sbin/logrotate {{ home }}/poa-logrotate.conf -s /home/bootnode/logrotate.status" user={{ username }} name="logrotate"
```

Comment:
Seems state file option should be `{{ home }}/logrotate.status`.
Also let's give it name `poa-logrotate`, current name look too general